components/infobox/wikis/mobilelegends/infobox_hero_custom.lua

```diff
@@ -34,6 +35,22 @@ local _pagename = mw.title.getCurrentTitle().text
 local _BATTLE_POINTS_ICON = '[[File:Mobile_Legends_BP_icon.png|x16px|Battle Points|link=Battle Point]]'
 local _DIAMONDS_ICON = '[[File:Mobile_Legends_Diamond_icon.png|Diamonds|x16px|link=Diamond]]'
 
+local NON_BREAKING_SPACE  = '&nbsp;'
+
+function CustomHero._voiceActors()
```

Comment:
imo move it further down
the run imo should be the first function to see as it is the one that gets invoked